lib/generators/manageiq/provider/templates/app/models/%plugin_path%/%manager_path%.rb

```diff
@@ -140,4 +141,61 @@ def self.ems_type
   def self.description
     @description ||= "<%= provider_name.split('_').map(&:capitalize).join(' ') %>".freeze
   end
+
+  # TODO: This class represents a fake Ruby SDK with sample data.
+  #       Remove this and use a real Ruby SDK in the raw_connect method
+  class MyRubySDK
+    def vms
+      [
+        OpenStruct.new(
+          :id       => '1',
+          :name     => 'funky',
+          :location => 'dc-1',
+          :vendor   => 'unknown'
+        ),
+        OpenStruct.new(
+          :id       => '2',
+          :name     => 'bunch',
+          :location => 'dc-1',
+          :vendor   => 'unknown'
+        ),
+      ]
+    end
+
+    def find_vm(id)
+      vms.find { |v| v.id == id.to_s }
+    end
+
+    def events
+      [
+        OpenStruct.new(
+          :name       => %w(instance_power_on instance_power_off).sample,
+          :id         => Time.zone.now.to_i,
+          :timestamp  => Time.zone.now,
+          :vm_ems_ref => [1, 2].sample
```

Comment:
I think it's weird the "fake data" has ems_ref in it, but wasn't sure how the events scaffolding expected this data to work.  I thought we usually have an events parser somewhere.

Reply:
Looks like this generator doesn't build a typical event parser class, but instead has that "built-in" to the event catcher runner, and passes through the `vm_ems_ref`.

Not for this PR but given the inventory collector here maps `#id` => `#ems_ref` it'd make more sense for the event to have `#vm_id` and an event parser mapping that to `#vm_ems_ref` with a proper event parser class.